dev/Deployment/Deployment.idl

```diff
@@ -5,7 +5,6 @@
 
 namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
 {
-    [feature(Feature_DeploymentRepair)]
     [contractversion(4)]
```

Comment:
Does the [feature] tag need to be applied to other types that were added in v4?

Reply:
The 2 types new in v4 were already tagged (line 20+76). There's a separate thread discussing if that should change for 1.4 or (more likely) 1.5, but it's not a problem. [feature] on apicontract was the surprise build breaker.